charts/internal/seed-controlplane/charts/csi-alicloud/templates/csi-plugin-controller.yaml

```diff
@@ -30,7 +30,7 @@ spec:
         networking.gardener.cloud/to-alicloud-networks: allowed
     spec:
       containers:
-      - name: csi-diskplugin
+      - name: alicloud-csi-diskplugin
```

Comment:
why do we need change this container name?

Reply:
Yes, we need this change, because fluent-bit should have unique Filter matchers